mautic/mautic
# New Campaign actions:  Add/Remove Do Not Contact

**Please be sure you are submitting this against the _staging_ branch.**

[//]: # ( Please answer the following questions: )

| Q  | A
| --- | ---
| Bug fix? | 
| New feature? | 
| Automated tests included? |
| Related user documentation PR URL | 
| Related developer documentation PR URL | 
| Issues addressed (#s or URLs) | 
| BC breaks? | 
| Deprecations? | 

[//]: # ( Note that all new features should have a related user and/or developer documentation PR in their respective repositories. )

[//]: # ( Required: )
#### Description:
This PR added Add/Remove DNC as campaign actions

![image](https://user-images.githubusercontent.com/462477/58506352-6b2e1780-818f-11e9-8c39-afd7303c8dea.png)

![image](https://user-images.githubusercontent.com/462477/58506305-49cd2b80-818f-11e9-8c0b-214c89f4fccf.png)

#### Steps to test this PR:
1. Load up [this PR](https://mautibox.com)
2. Create campaign and add contacts
3. Set Add Do Not Contact action
4. See If contacts were marked as DNC
5. Remove action and add new action: Remove Do Not Contact
6. See If contacts were unmarked as DNC
